### ISSUE TITLE
Some a11y issues

### ISSUE THREAD

**Issue by jrock2004** [empress/empress-blog-casper-template#35]
So I am not sure if this is something that should fall on us to just override in our own CSS file or if this is something that should be fixed by the template itself. These can be discovered by running axe plugin in Firefox

- Date and tag above Post title fails color contrast
- Read more posts by this author fails color contrast
- Read more button on the bottom fails color contrast
- Length of time to read fails color contrast
- Floating header title color contrast
- Floating share links to facebook and twitter are missing descriptive text
- site-nav landmarks appear twice
